libs/src/sdk/oauth/OAuth.ts

```diff
@@ -123,6 +126,7 @@ export class OAuth {
     this.loginSuccessCallback = null
     this.loginErrorCallback = null
     this.popupCheckInterval = null
+    this.localStorage = new LocalStorage({ localStorage: window.localStorage })
```

Comment:
Correct me if I'm wrong, but SDK can run in more places than just web/browser? If so,  I believe `window` wouldn't be safe to access without a check.